packages/api-client/src/connections/MeditrakApi.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+
+import { BaseApi } from './BaseApi';
+
+export type SurveyResponse = {
+  surveyId: string;
+  entityCode: string;
+  timestamp: string;
+  answers: Answers;
+}
+
+export type Answers = {
+  [key: string]: string; // question_code -> value
+}
+
+export class MeditrakApi extends BaseApi {
+
+  async createSurveyResponses(responses: SurveyResponse[]): Promise<void> {
```

Comment:
The existing MeditrakConnections we could have used use the postChanges endpoint, which requires generateId() etc. to be done before the request is made to Meditrak. And then we would have had to copy those functions into this package for publishing.

I decided on using POST /surveyResponse because it didn't have these extra requirements.

It seems both API endpoints are in active development so it seems there is a choice to choose between the two.

This is why this class looks very empty.